src/resources/postcss/tickets.pcss

```diff
@@ -303,10 +303,6 @@
 		}
 	}
 
-	.tribe-tickets__item__quantity__add {
```

Comment:
Was removing this intentional? Didn't see a changelog note about it

Reply:
Yes -> https://github.com/moderntribe/event-tickets/pull/1628/commits/02a70ed829f98da5a7dd780ad947a31281452c51